src/main/Main.cpp

```diff
@@ -178,12 +179,12 @@ try
     io_context ioc{threads};
 
     // Rate limiter, to prevent abuse
-    auto sweepHandler = IntervalSweepHandler{config, ioc};
-    auto whitelistHandler = WhitelistHandler{config};
-    auto dosGuard = DOSGuard{config, whitelistHandler, sweepHandler};
+    auto sweepHandler = web::IntervalSweepHandler{config, ioc};
+    auto whitelistHandler = web::detail::WhitelistHandler{config};
```

Comment:
I'd avoid detail namespace for WhitelistHandler if we are using it here in main.